v21.2/architecture/storage-layer.md

```diff
@@ -51,6 +51,80 @@ Pebble integrates well with CockroachDB for a number of reasons:
 
 Efficient storage for the keys is guaranteed by the underlying Pebble engine by means of prefix compression.
 
+For more information about Pebble, see the [Pebble GitHub page](https://github.com/cockroachdb/pebble) or the blog post [Introducing Pebble: A RocksDB Inspired Key-Value Store Written in Go](https://www.cockroachlabs.com/blog/pebble-rocksdb-kv-store/).
+
+Pebble uses a log structured merge tree (_LSM_) to manage data storage. For more information about how LSM-based storage engines like Pebble work, see [Log structured merge trees](#log-structured-merge-trees) below.
+
+#### Log structured merge trees
+
+Pebble uses a Log structured merge tree (hereafter _LSM tree_ or _LSM_) to manage data storage. The LSM is a hierarchical tree. At each level of the tree, there are files on disk that store the data referenced at that level. The files are known as _sorted string table_ files (hereafter _SST_ or _SST file_).
+
+##### SSTs
+
+SSTs are an on-disk representation of sorted lists of key-value pairs. Conceptually, they look something like this:
+
+<img src="{{ 'images/v21.2/sst.png' | relative_url }}" alt="Structure of an SST file" style="max-width:100%" />
+
+SST files are immutable; they are never modified, even during the [compaction process](#compaction). 
+
+##### LSM levels
+
+The levels of the LSM are organized from L0 to L6. L0 is the top-most level. L6 is the bottom-most level. New data is ingested into L0 (e.g. using [`INSERT`](../insert.html) or [`IMPORT`](../import.html)) and then merged down into lower levels over time.
```

Comment:
`s/ingested/added` in this spot - thanks!  

Since you did not mention the other two uses of "ingest*" on this page should I assume they are used correctly?  Or would it be better if those were also changed to "added"?

Reply:
Changed the one on line 95 to "added"!